src/main/java/rsb/methods/Game.java

```diff
@@ -64,57 +65,52 @@ public String getLastMessage() {
 	 * @param tab The tab to open.
 	 * @return <code>true</code> if tab successfully selected; otherwise
 	 *         <code>false</code>.
-	 * @see #openTab(GameGUI.Tab tab, boolean functionKey)
+	 * @see #openTab(InterfaceTab tab, boolean functionKey)
 	 */
-	public boolean openTab(GameGUI.Tab tab) {
+	public boolean openTab(InterfaceTab tab) {
 		return openTab(tab, false);
 	}
 
 	/**
 	 * Opens the specified tab at the specified index.
 	 *
-	 * @param tab 				The tab to open
-	 * @param functionKey       If wanting to use function keys to switch.
+	 * @param interfaceTab The tab to open
+	 * @param useHotkey If wanting to use hotkeys to switch.
 	 * @return <code>true</code> if tab successfully selected; otherwise
 	 *         <code>false</code>.
 	 */
-	public boolean openTab(GameGUI.Tab tab, boolean functionKey) {
+	public boolean openTab(InterfaceTab interfaceTab, boolean useHotkey) {
 		// Check current tab
-		if (tab == getCurrentTab()) {
-			return true;
-		}
-
-		if (functionKey) {
-			if (tab.getFunctionKey() == 0) {
-				return false;// no function key for specified tab
-			}
-
-			methods.keyboard.pressKey((char) tab.getFunctionKey());
+		if (interfaceTab == getCurrentTab()) { return true; }
+		if (useHotkey) {
+			if (interfaceTab.getHotkey() == 0) { return false; } // no hotkey for specified tab
+			methods.keyboard.pressKey((char) interfaceTab.getHotkey());
 			sleep(random(80, 200));
-			methods.keyboard.releaseKey((char) tab.getFunctionKey());
+			methods.keyboard.releaseKey((char) interfaceTab.getHotkey());
```

Comment:
Add a check to see if the hotkey press worked (for same reason as above), and print something if it failed - if it fails, it should go through with a click

(A better solution to all of this is you could try to see which Varp controls the hotkey)